projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -1,12 +1,37 @@
 import { Component, Element, Event, EventEmitter, Listen, Prop, State } from '@stencil/core';
-import { DotCMSContentTypeField, DotFieldStatus } from '../../models';
+import { DotCMSContentTypeField, DotFieldStatus, DotCMSKeyValueField } from '../../models';
 import { DotFormFields } from './dot-form-fields';
 import { getClassNames, getOriginalStatus } from '../../utils';
 
+const fieldParamsConversionToBE = {
+    'Key-Value': (values: string): { [key: string]: string } => {
+        const returnValue = {};
+        values.split(',').forEach((item: string) => {
```

Comment:
This code and [this](https://github.com/dotCMS/core-web/pull/1008/files#diff-3923e7309ccdea946eb1f2dc46e564e7R147) is the same, can we move this to their own function helper?

Reply:
they're different:
1) https://github.com/dotCMS/core-web/blob/96894a92b2299c428f66d30fdf342146475e256a/projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx#L9
returns an Object ``` {a: 1, b:2, c:3}```

2) https://github.com/dotCMS/core-web/blob/aaf49cf2d22d3f9d4ff1affdeaf9878806934cbe/projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.tsx#L141
returns an array of objects ```[ {a: 1}, {b: 2}, {c: 3} ]```